compiler/noirc_evaluator/src/ssa/ir/dfg.rs

```diff
@@ -100,6 +105,34 @@ pub(crate) struct DataFlowGraph {
 }
 
 impl DataFlowGraph {
+    pub(crate) fn new(runtime: RuntimeType) -> Self {
+        Self {
+            runtime,
+            instructions: Default::default(),
+            results: Default::default(),
+            values: Default::default(),
+            constants: Default::default(),
+            functions: Default::default(),
+            intrinsics: Default::default(),
+            foreign_functions: Default::default(),
+            blocks: Default::default(),
+            replaced_value_ids: Default::default(),
+            locations: Default::default(),
+            call_stack_data: Default::default(),
+            data_bus: Default::default(),
```

Comment:
```suggestion
            ..Default::default()
```
Should be able to just do this

Reply:
Ah yes, I just accepted the editor filling it in without thinking 🤦‍♂️ Thanks.

Reply:
Hm, no, `Runtime` doesn't have a `Default`, so even if I can't add back `#[derive(Default)]` to `DataFlowGraph` to enable this syntax.

Reply:
We could simply add a default for `RuntimeType` (we do this for `InlineType`). But I won't block on this as this is a minor syntax choice.

Reply:
Okay, added it! I suppose it was the default anyway by the virtue of `Function::new` assigning a default value to its own field.